src/System.ComponentModel.TypeConverter/tests/System.ComponentModel.TypeConverter.Tests.csproj

```diff
@@ -159,5 +167,15 @@
       <LastGenOutput>TestResx.Designer.cs</LastGenOutput>
     </EmbeddedResource>
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <SupplementalTestData Include="$(PackagesDir)system.componentmodel.typeconverter.testdata\1.0.0\content\**\*.*">
+      <Link>%(RecursiveDir)%(Filename)%(Extension)</Link>
+    </SupplementalTestData>
+    <EmbeddedResource Include="$(PackagesDir)system.componentmodel.typeconverter.testdata\1.0.0\content\bitmaps\TestIcon.ico">
```

Comment:
You can delete this EmbeddedResources and just load the images from file as it is faster. You can just use the files from the bitmap folder since they are copied by the SupplementalTestData item that you added above. Then your Icon and Image test constructors would look something like this:

```cs
public ImageConverterTest()
{
    _image = Image.FromFile(Path.Combine("bitmaps", "TestImage.bmp"));
    _imageStr = _image.ToString();

    using (MemoryStream destStream = new MemoryStream())
    {
        _image.Save(destStream, _image.RawFormat);
        _imageBytes = destStream.ToArray();
    }

    _imgConv = new ImageConverter();
    _imgConvFrmTD = (ImageConverter)TypeDescriptor.GetConverter(_image);
}
```

and for IconTests something similar but using `Icon..ctor(string fileName)`

and btw this fixes your test failures since we're now relying on Image.Save to get the image bytes.